src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageItemControl.xaml.cs

```diff
@@ -15,5 +19,47 @@ public PackageItemControl()
         {
             InitializeComponent();
         }
+
+        // Whenever the checkbox in the item is checked, the containing item will raise a 
+        // UIA event to convey the fact that the toggle state of the item has changed. The 
+        // event must be raised regardless of whether the toggle state of the item changed 
+        // in response to keyboard, mouse, or programmatic input.
+        private void CheckBox_Checked(object sender, RoutedEventArgs e)
+        {
+            var itemCheckBox = sender as CheckBox;
+            if (itemCheckBox is null || itemCheckBox.Visibility != Visibility.Visible)
+            {
+                return;
+            }
+
+            var parentContentPresenter = VisualTreeHelper.GetParent(this) as ContentPresenter;
+            if (parentContentPresenter is null)
+            {
+                return;
+            }
+
+            var parentBorder = VisualTreeHelper.GetParent(parentContentPresenter) as Border;
+            if (parentBorder is null)
+            {
+                return;
+            }
+
+            var itemContainer = VisualTreeHelper.GetParent(parentBorder) as ListBoxItem;
+            if (itemContainer is null)
+            {
+                return;
+            }
+
+            var isChecked = (e.RoutedEvent == CheckBox.CheckedEvent);
```

Comment:
Parentheses can be removed.

Reply:
That's not a coding guideline violation I'm aware of. I think it's more readable in this case.

Reply:
It violates [IDE0047](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0047-ide0048)

Reply:
That's a nice find! The way I read it though, `dotnet_style_parentheses_in_relational_binary_operators` default value is `always_for_clarity` which gives an example with parenthesis similar to this code (ex: `var v = (a < b) == (c > d);`)